README.pod

```diff
@@ -63,6 +63,27 @@ a subset of the functionality).
 It is based on Martin Berends' MiniDBI project, but unlike MiniDBI, DBDish
 aims to provide an interface that takes advantage of Perl 6 idioms
 
+=head2 Connecting to, and disconnecting from, a database
+
+For a database named I<hierarchy>, a connection may be established with any of
+the following syntaxes:
+
+    my ($dbh, $db);
+
+    $dbh = DBIish.connect("Pg", :database<hierarchy>);
+    $dbh.dispose;
+
+    $db = 'hierarchy';
+
+    $dbh = DBIish.connect("Pg", :database($db));
+    $dbh.dispose;
+
+    $dbh = DBIish.connect("Pg", database => $db);
+    $dbh.dispose;
+
+    $dbh = DBIish.connect("Pg", :$db);
```

Comment:
@jkeenan, do you have a `PGDATABASE` environment variable set? That can explain why worked for you.
Not using any of `:database` nor its alias `:dbname` makes the Pg driver fallback to search the environment.